include/NovelRT/Experimental/Graphics/GraphicsPipelineBlendFactor.h

```diff
@@ -0,0 +1,37 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root
+// for more information.
+
+#ifndef NOVELRT_EXPERIMENTAL_GRAPHICS_GRAPHICSPIPELINEBLENDFACTOR_H
+#define NOVELRT_EXPERIMENTAL_GRAPHICS_GRAPHICSPIPELINEBLENDFACTOR_H
+
+#ifndef NOVELRT_EXPERIMENTAL_GRAPHICS_H
+#error NovelRT does not support including types explicitly by default. Please include Graphics.h instead for the Graphics namespace subset.
+#endif
+
+namespace NovelRT::Experimental::Graphics
+{
+    enum class GraphicsPipelineBlendFactor : uint32_t
```

Comment:
nit: It looks like you just copied `VkBlendFactor` (https://www.khronos.org/registry/vulkan/specs/1.2-extensions/man/html/VkBlendFactor.html). This is going to be non-portable if we add `D3D12` later and it might risk introducing bugs if we just assume blittable casts between our enum and Vks enum

Reply:
I have opened an issue, #391 , to address this. For now I am resolving as we do not have D3D12 currently.